mods/cnc/chrome/ingame.yaml

```diff
@@ -1549,7 +1549,7 @@ Container@PLAYER_WIDGETS:
 			X: WINDOW_RIGHT - WIDTH - 5
 			Y: 5
 			Width: 230
-			Height: 291
+			Height: 310
```

Comment:
```suggestion
			Height: 314
```

The element below is at 319. So to remove all gamps it needs to be 319 - 5 = 314

Reply:
Thank you for looking! That was intentionally, because it‘s at an angle at the end. But we can use your value if you rather want it like this. So, what do you want?

Reply:
What's at an angle? Also as I didn't notice any of the visuals changing I think it's for the best to keep it the proper size. I even tried setting it to 100 and everything looked the same

Reply:
I‘ll test tomorrow and make some screenshots! Thanks for your sharp eyes!

Reply:
@ThomasChr We can't describe the clickable area as a polygon and that corner is fairly small so I say ignore it and go for 314.

Reply:
Used 314, alright!